assets/js/autocomplete/suggestions.js

```diff
@@ -133,12 +145,20 @@ function childNodeSuggestion (childNode, parentId, query, category, label) {
 function nodeSectionSuggestion (node, section, query, category, label) {
   if (!matchesAny(section.id, query)) { return null }
 
+  let link
+
+  if (section.anchor === '' || !section.anchor) {
+    link = `${node.id}.html`
+  } else {
+    link = `${node.id}.html#${section.anchor}`
+  }
+
   return {
-    link: `${node.id}.html#${section.anchor}`,
+    link,
     title: highlightMatches(section.id, query),
-    description: node.title,
+    description: section.description || node.title,
```

Comment:
This conditional is necessary because search items may customize the text below the title, but currently extra sections use their document's name (`node.title`).

Reply:
Remember that our goal is to unify this with TypeSense search, as the TypeSense search will be the default in the future for Ash most likely. And this description field is not indexed in TypeSense at all, so this feature will be lost anyway. I suggest removing it then.

Reply:
Hmm... regardless of how things change, however, the autocomplete would look "bad" without the description I'm providing. I.e it would either be the title of the document or nothing. For now, I'll remove because it is always easier to add to an API than remove, and I can revisit if needed. Thinking about it, the title of the extra being the description isn't the worst thing ever. 👍🏻